central/node/datastore/datastore.go

```diff
@@ -103,3 +104,8 @@ func GetTestRocksBleveDataStore(t testing.TB, rocksengine *rocksdbBase.RocksDB,
 	nodeComponentRanker := ranking.NodeComponentRanker()
 	return New(dacky, keyFence, bleveIndex, riskStore, nodeRanker, nodeComponentRanker), nil
 }
+
+// NodeString returns a human-readable string representation of a node.
+func NodeString(node *storage.Node) string {
+	return fmt.Sprintf("%s:%s (id: %s)", node.GetClusterName(), node.GetName(), node.GetId())
+}
```

Comment:
I've added this here because my initial feeling is that this logic should be close to its model, and I could not find a better candidate. Please correct me if you know a better place.

Reply:
`pkg/node`